gnd/src/main/java/com/google/android/gnd/ui/editrecord/EditRecordViewModel.java

```diff
@@ -64,6 +70,8 @@
   private final Resources resources;
   private final ObservableMap<String, Response> responses = new ObservableArrayMap<>();
   private final ObservableMap<String, String> errors = new ObservableArrayMap<>();
+  private final BehaviorProcessor<EditRecordFragmentArgs> argumentProcessor;
+  private final BehaviorProcessor<Record> recordProcessor;
```

Comment:
Same comment as before, please name streams for what emitted items represent, if possible. So `argumentProcessor` might be requests to edit the record (`editRecordRequests`) and `recordProcessor` would be `saveRecordRequests` or something similar.

Reply:
Also, not sure that Behaviors are correct here; I think Publishers might be more appropriate?